starboard/socket_waiter.h

```diff
@@ -79,6 +79,14 @@ typedef void (*SbSocketWaiterCallback)(SbSocketWaiter waiter,
                                        void* context,
                                        int ready_interests);
 
+#if SB_API_VERSION >= 16
+// Function pointer for socket waiter callbacks.
+typedef void (*PosixSocketWaiterCallback)(SbSocketWaiter waiter,
```

Comment:
Those need to be `SbPosixSocketWaiterCallback` etc... 

All the SB APIs start with `Sb`.